styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -109,7 +109,7 @@ private BackfillsPayload getBackfills(RequestContext requestContext) {
 
     Stream<Backfill> backfills;
     try {
-      backfills = storage.backfills().stream();
+      backfills = storage.backfills().parallelStream();
```

Comment:
Could this be moved to where the intensive part is? It doesn't really matter for the filtering, right? We're even moving that to datastore at some point.